migrations/versions/18ecb660e0dc_books.py

```diff
@@ -0,0 +1,26 @@
+"""books
```

Comment:
Z migracjami się tak nie pracuje, ale to już wam odpuszczę. Następnym razem jak wam porobią się takie puste pliki to:
- usuńcie plik bazy .sql
- usuńcie katalog migrations
- python manage.py db init
- python manage.py db migrate -m "jakiś komętarz"
- python manage.py db upgrade